src/core/opamExternalTools.ml

```diff
@@ -0,0 +1,125 @@
+(**************************************************************************)
+(*                                                                        *)
+(*    Copyright 2018 OCamlPro                                             *)
```

Comment:
this isn't copyright ocamlpro :)

Reply:
I was just copying the header from the other files. Should I put my name instead ?

Reply:
Actually, in this instance I think it depends - is the module predominantly just refactoring?

Reply:
yes, I think so.